src/3p.js

```diff
@@ -114,7 +114,7 @@ export function validateSrcPrefix(prefix, src) {
     prefix = [prefix];
   }
 
-  for (const p of prefix) {
+  for (const p in prefix) {
```

Comment:
There are additional issues to address with this method, please see Issue #2007 for more information. I will attempt resolve it with this branch unless your team is already aware of the issue and there is another fix under development.